### ISSUE TITLE
2.7.0 PostgreSQL Issue

### ISSUE THREAD

**Issue by rChassat** [RamyHakam/multi_tenancy_bundle#42]
Hello 👋

This changes in 2.7.0 to the DbService class caused an issue with PostgreSql db config :

```php
$dsnParser = new DsnParser(['mysql' => 'pdo_mysql']);
$tmpConnection = DriverManager::getConnection($dsnParser->parse($this->dbCredentials['db_url']));
```

You should probably add all the supported entries in the array passed to the DsnParser constructor or direcly use `Doctrine\Bundle\DoctrineBundle\ConnectionFactory::DEFAULT_SCHEME_MAP`  array.

**Comment by rChassat** [comment]
Just saw that there is this pull request #38  that also concerns that issue.

**Comment by RamyHakam** [comment]
Thanks for your support and help, The fix has been released

**Comment by RamyHakam** [comment]
It is fixed in 2.7.1
